dependencies/pom.xml

```diff
@@ -56,7 +56,7 @@
         <version.lib.google-api-client>1.34.1</version.lib.google-api-client>
         <version.lib.google-oauth-client>1.33.3</version.lib.google-oauth-client>
         <version.lib.google-error-prone>2.3.3</version.lib.google-error-prone>
-        <version.lib.google-protobuf>3.19.2</version.lib.google-protobuf>
+        <version.lib.google-protobuf>3.21.4</version.lib.google-protobuf>
```

Comment:
If this version is only used for the version of `protoc` then consider renaming it to `version.lib.protoc`.

Reply:
It is also used to explicitly manage protobuf:
```xml
          <dependency>
              <groupId>com.google.protobuf</groupId>
              <artifactId>protobuf-java</artifactId>
              <version>${version.lib.google-protobuf}</version>
          </dependency>
```